### DISCUSSION TITLE
Images attached to tickets are using HTTP instead of HTTPS

### DISCUSSION THREAD

**Discussion by slaakso** [cerb/cerb-release#55] [Q&A] [answered]
Hi,
when a ticket comes in with an image attachment and it is being opened in Cerb with https, the image URL is using http instead of https. This causes Safari to blocks the image loading due to Content Security Policy.

Any other way of fixing this other than modifying the code? Any setting/config?

**Comment by Hildy** [comment, marked as answer]
What protocol is your cron job using?  I suspect you're using HTTP for the cron connection, which is what generates those URLs.  If so, switch that to HTTPS.  (Run it manually from the command line to make sure you don't have any certificate issues first, though.)

**Reply by slaakso** [reply]
That was it. Thanks.
